libqtile/widget/net.py

```diff
@@ -30,6 +30,8 @@ class Net(base.ThreadedPollText):
     """Displays interface down and up speed"""
     orientations = base.ORIENTATION_HORIZONTAL
     defaults = [
+        ('display_format', '{interface}: {down} \u2193\u2191 {up}',
```

Comment:
Let's stay consistent with other widgets:

```suggestion
        ('format', '{interface}: {down} \u2193\u2191 {up}',
```

Reply:
Sure I don't mind that, although I did copy the naming from the check_updates widget for consistency, see https://github.com/qtile/qtile/blob/f186e93a6ae5e23d17f1d177e226eaa123b27cdb/libqtile/widget/check_updates.py#L35